api/openapi/model-registry.yaml

```diff
@@ -1224,9 +1224,8 @@ components:
           properties:
             id:
               format: int64
-              description: Output only. The unique server generated id of the resource.
+              description: The unique server generated id of the resource.
               type: string
-              readOnly: true
```

Comment:
I'm not sure I understood the reason for this change?
iirc we rely on unique id generated on MLMD side.

Reply:
I tried to summarize in the commit description, but I can go into more detail:
Because we're using some vanilla models (i.e. not `*Create` or `*Update`) in routes that support `upsert`, we need to be able to send the model id in the request.
If we don't expose this field, we would need to incorrectly patch the Python autogen bindings, violating the openapi spec. Now, every client would need to do that, or we could create new models for the upsert routes that have this field exposed, but then the clients would lose on the simplicity of doing an upsert e.g. would have to convert to an intermediate model with no benefit.